packages/internal-playwright-helpers/src/pages/cognito.ts

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { Page } from "@playwright/test";
+
+/**
+ * Login page exposed by Cognito, the OIDC Provider used for PodSpaces and other
+ * ESS deployments.
+ */
+export class CognitoPage {
+  page: Page;
+
+  static COGNITO_DOMAIN = "amazoncognito.com ";
```

Comment:
My gut is telling me we shouldnt be hardcoding this, but its consistent across all envs so there is no real reason to right now.

Reply:
Well it's not great, but since the whole concept of `pages` is pretty much hardcoding what we expect of the pages at least it kinda belongs there I guess